event.c

```diff
@@ -1755,6 +1817,16 @@ event_process_active_single_queue(struct event_base *base,
 		}
 #endif
 
+#ifdef EVENT__HAVE_LIBURING
+		if (base->io_uring_sqe_count) {
```

Comment:
Hm, interface looks clumsy:
- checking `io_uring_sqe_count` after each event looks odd, it should be checked in the `event_base_loop`, or it should be binded to specific event and processed only when the callback for that particular event is called (one way is to add new callback for event in case of uring, however a) this will bloat the structure and b) it will break ABI compatibility - even though it is highly not recommended to use `event` structure directly)
- `io_uring_sqe_count` is set from the `buffer.c`, and checked here, how about adding some wrappers for this?
- `io_uring_buffers` modified directly from the `bufferevent` code, how about adding some wrappers for this?